output/elasticsearch/fluent-bit-ds(v1.16+).yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: fluent-bit
+  namespace: logging
+  labels:
+    k8s-app: fluent-bit-logging
+    version: v1
+    kubernetes.io/cluster-service: "true"
+spec:
+  selectors:
```

Comment:
FYI, this line gave me an error. I had to change it to the singular form 'selector'.
```
[ValidationError(DaemonSet.spec): unknown field "selectors" in io.k8s.api.apps.v1.
DaemonSetSpec, ValidationError(DaemonSet.spec): missing required field "selector" in io.k8s.api.apps.v1.DaemonSetSpec]
```

Reply:
thanks for pointing out.